tests/binaries/host/makefile

```diff
@@ -69,19 +68,11 @@ heap_bugs.out: heap_bugs.c
 	${ZIGCC} \
 	${CFLAGS} \
 	-Wno-int-to-pointer-cast -Wno-int-conversion -Wno-unused-variable \
-	-target native-native-gnu.2.33 \
-	-Wl,-rpath=${GLIBC_2_33}:\
-	-Wl,--dynamic-linker=${GLIBC_2_33}/ld-linux-x86-64.so.2 \
```

Comment:
This was here for a good reason. We test our heap features against specific versions of `glibc`.